showcase/src/app/welcome/pages/home/home-page.component.html

```diff
@@ -1,40 +1,85 @@
-<span class="stark-section about-background"></span>
-<p class="mat-body-2 light-font" [innerHTML]="'SHOWCASE.HOMEPAGE.DESCRIPTION_BEFORE_DETAIL' | translate"></p>
-<p class="mat-h4" [innerHTML]="'SHOWCASE.HOMEPAGE.DESCRIPTION_DETAIL' | translate"></p>
-<p class="mat-body-2 light-font" [innerHTML]="'SHOWCASE.HOMEPAGE.DESCRIPTION_DETAIL_LEGO_BLOCKS' | translate"></p>
-<a class="github-link" href="https://github.com/NationalBankBelgium" target="_blank" rel="noopener noreferrer">
-	<mat-icon class="github-icon" svgIcon="github-circle" starkSvgViewBox></mat-icon>
-</a>
-<p class="mat-body-2 light-font" [innerHTML]="'SHOWCASE.HOMEPAGE.DESCRIPTION_AFTER_DETAIL' | translate"></p>
+<div class="section-division">
+	<div class="stark-logo"></div>
+	<p class="centered-title" [innerHTML]="'SHOWCASE.HOMEPAGE.INTRODUCTION' | translate"></p>
+	<div class="buttons-row">
+		<a class="button custom-buttons"
```

Comment:
This button should navigate to the "GettingStarted" page instead of opening the Showcase/GettingStarter in a new browser tab... so you should use the `sref` directive as we do in the App Menu ;)